### PR TITLE
Closes #402: Improve release process

### LINKED ISSUE #402: Create document that outlines release process
similar too https://github.com/gnosis/safe-ios/blob/master/RELEASE.md

add rinkeby version to crashlytics

### PR DESCRIPTION
**Pull request by rmeissner**
Closes #402

Changes proposed in this pull request:
- Add release instructions
- Upload rinkeby version to beta

@gnosis/mobile-devs

### CODE REVIEW

File: docs/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Release Process
+
+## QA
+
+1. Decide what commit should be used for the release and create a draft release
+  - The description should contain a list of new features
+2. Share release and a Crashlytics Beta link to that commit with the QA team
+  - New features and core features (create Safe, send transactions, restore Safe) MUST be tested with the Mainnet version
```
Comment (elgatovital) [inline docs/RELEASE.md]: should link to qa google doc be linked here?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,7 @@
...
-
 ## Goal
-The goal of this project is to provide a 2FA-enabled Ethereum Wallet. This can be achieved by using the new [Gnosis Safe](https://github.com/gnosis/gnosis-safe-contracts) (the new version of our [Multisig Wallet](https://github.com/gnosis/MultiSigWallet)). 
-
-You can add multiple devices to the safe and secure your wallet by requiring a set of devices to confirm each transaction. Depending on you security setting and number of devices you can still access your funds even if you loose access to a device.
-
-Deploying and interacting with your Gnosis Safe will also be much cheaper than with our Multisig Wallet.
-
-## Roadmap
-### Current Features
```
Comment (elgatovital) [inline README.md]: should we leave/update list of implemented features?

File: docs/RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Release Process
+
+## QA
+
+1. Decide what commit should be used for the release and create a draft release
```
Comment (elgatovital) [inline docs/RELEASE.md]: maybe explicitly specify that only master branch commits are eligible for release

File: docs/RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+2. Share release and a Crashlytics Beta link to that commit with the QA team
+  - New features and core features (create Safe, send transactions, restore Safe) MUST be tested with the Mainnet version
+  - Other features SHOULD be tested with the Rinkeby version
+3. If required fix bugs and repeat process
+
+## Build release version
+1. Finalize the draft created in [QA step](#qa)
+  - CI will automatically build a Mainnet and Rinkeby versions, which will be attached to the release
+2. Download Mainnet apk file (prefixed with `-release`)
+3. Create new version in PlayStore with downloaded Mainnet apk file
```
Comment (elgatovital) [inline docs/RELEASE.md]: Who is responsible for writing release notes?

File: README.md (hunk 2 above)
Comment (rmeissner) [reply README.md]: We can add this again in the future ... but this was just wrong and therefore I removed it

File: docs/RELEASE.md (hunk 3 above)
Comment (rmeissner) [reply docs/RELEASE.md]: no necessarily :P (normally it is, but hotfixes would not always be on the `master` branch for example)

I will add some info

File: docs/RELEASE.md (hunk 4 above)
Comment (rmeissner) [reply docs/RELEASE.md]: For the play store? I will add some more info

File: docs/RELEASE.md (hunk 1 above)
Comment (rmeissner) [reply docs/RELEASE.md]: no I would not link any private docs here

File: docs/RELEASE.md (hunk 4 above)
Comment (elgatovital) [reply docs/RELEASE.md]: yes

File: docs/RELEASE.md (hunk 3 above)
Comment (elgatovital) [reply docs/RELEASE.md]: every release should be on the master branch. Don't we merge hofix-branches to master before releasing?

File: README.md [hunk 5]
```diff
@@ -1,31 +1,7 @@
...
-* Fiat prices for transaction costs (USD)
-* Fingerprint support
-* Add/Remove owner from deployed safes
-
-### To be added
-* Trezor support
-* Transaction scanning with QRCode
-* Integration with external protocols/apps for signing
-
+The goal of this project is to provide a 2FA-enabled Ethereum Wallet. This can be achieved by using the new [Gnosis Safe](https://github.com/gnosis/safe-contracts) (the new version of our [Multisig Wallet](https://github.com/gnosis/MultiSigWallet)).
```
Comment (tschubotz) [inline README.md]: I know it's a page for technical users, but I'd reuse some of the things we already have. How about the following text:
```
The Gnosis Safe aims to provide all users with a convenient, yet secure way to manage their funds and interact with decentralized applications on Ethereum. 
Transactions are secure by on-chain 2-factor-authentication. This is achieved by using the [Gnosis Safe smart contracts](https://github.com/gnosis/safe-contracts). They are the successor of the broadly trusted [Gnosis Multisig Wallet smart contracts](https://github.com/gnosis/MultiSigWallet)).
```

File: docs/RELEASE.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  - Check with communications and product team for PlayStore assets
+    - Version release notes
+    - Updates to screenshots or store description
+  - Check if there should be an announcement for the new release
```
Comment (tschubotz) [inline docs/RELEASE.md]: Do you think we should add 2 more steps?
```
## Release
## Monitor stability via Crashlytics
```

Comment (rmeissner) [reply docs/RELEASE.md]: what would you add for `Release`?

**Pull request merged**